kernel/src/fs/configfs/mod.rs

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use alloc::sync::Arc;
+
+use aster_systree::SysBranchNode;
+use fs::ConfigFs;
+use spin::Once;
+use systree_node::ConfigRootNode;
+
+use crate::fs::configfs::fs::ConfigFsType;
+
+mod fs;
+mod inode;
+mod systree_node;
+#[cfg(ktest)]
+mod test;
+
+static CONFIG_SINGLETON: Once<Arc<ConfigFs>> = Once::new();
+
+/// Returns a reference to the global [`ConfigFs`] instance. Panics if not initialized.
```

Comment:
Be consistent with yourself.

The three functions in this module refer to the concept of ConfigFS in three different ways:

> [`ConfigFs`]

> ConfigFs`

> configfs

The docs about panics are not consistent, either.

In `register_subsystem`, you have correctly documented the panicking condition in the `Panics` section. But in this function, the panicking condition is given in the summary sentence, which is not a good style.